Detections/SlackAudit/SlackAuditUnknownUA.yaml

```diff
@@ -0,0 +1,31 @@
+id: 0ac695d3-6633-4e0b-a6a4-e89abc2737e1
+name: SlackAudit - Unknown User Agent
+description: |
+  'This query helps to detect who trying to connect to the Slack Workspace with Unknown User Agent.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: SlackAuditAPI
+    dataTypes:
+      - SlackAudit_CL
+queryFrequency: 24h
+queryPeriod: 60d
```

Comment:
14d max for lookback on detections to avoid potential performance issues

Reply:
Fixed, Pls check